app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryModel.kt

```diff
@@ -37,4 +37,8 @@ data class CustomDrugEntryModel(
   fun drugNameLoaded(drugName: String): CustomDrugEntryModel {
     return copy(drugName = drugName)
   }
+
+  fun rxNormCodeEdited(rxNormCOde: String?): CustomDrugEntryModel {
```

Comment:
Yes, because we are not tracking the drug object in the model and whenever we are updating or saving anything in the db we are creating a new entry. So, whenever we are fetching a prescription or a drug from their ids we are also saving the rxNormCode value so that when we save the medicine we save that value as well.

Reply:
Hmm, okay. I guess we can look if there is a better approach after the entire thing is implemented.